content/contest/hash-cpp.sh

```diff
@@ -0,0 +1 @@
+cpp -P -fpreprocessed | tr -d '[:space:]' | md5sum
```

Comment:
newline at end of file, and add a `cut`?

Reply:
add this to chapter.tex, with a description of how to use it and that it ignores comments and whitespace